src/container/container_iv.c

```diff
@@ -602,7 +604,8 @@ cont_iv_capa_refresh_ult(void *data)
 		D_GOTO(out, rc = -DER_NONEXIST);
 
 	rc = cont_iv_fetch(pool->sp_iv_ns, IV_CONT_CAPA,
-			   arg->cont_hdl_uuid, &entry, sizeof(entry));
+			   arg->cont_hdl_uuid, &entry, sizeof(entry),
+			   false /* retry */);
```

Comment:
so if the IV operation possibly happens during bootstrapping, then need to set retry flag as true?
so assume the cont_iv_capa_refresh_ult (now only used in obj_verify) is always after system fully bootstrapped?

Reply:
Hmm, it's not about bootstrapping. If the pool group changes, like adding/removing servers, IV operations may get -DER_MISMATCH. If this is originated from a client, we return the error and let the client retry; if this is originated from a server daemon ULT, we let the ULT retry. (The original initiator of the operation does the retries.)

Reply:
In the ULT retry case, Di suggested implementing the retry in ds_iv. So I added this flag.

Reply:
I see, thanks for explanations.